tasks/gem.rake

```diff
@@ -20,7 +20,7 @@ namespace :gem do
 
     s.required_ruby_version = '>= 2.0'
 
-    s.add_runtime_dependency 'public_suffix', '~> 2.0', '>= 2.0.2'
+    s.add_runtime_dependency 'public_suffix', '>= 2.0.2', '< 4.0'
```

Comment:
@houndci-bot should be consistent to all lines (and not only the affected ones), and is irrelevant for the change here.

(Yes I know you're a bot)